### PR TITLE
add massdrop/alt/hlmtre keymap, add .exe to gitignore

### PR DESCRIPTION
**Pull request by hlmtre**
## Description

Simply add a new keymap for myself. Very basic, unmaps caps and sets a few media keycodes to reasonable things, MO(1) + I becomes Insert.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,7 @@
 .history/
 .dep
 *.o
+*.exe
```
Comment (drashna) [inline .gitignore:4]: This should be pulled out and added to a different PR.  

Alternatively, you can use the global gitignore to not track exe's.

File: keyboards/massdrop/alt/keymaps/hlmtre/keymap.c [hunk 2]
```diff
@@ -0,0 +1,115 @@
+#include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/massdrop/alt/keymaps/hlmtre/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/massdrop/alt/keymaps/hlmtre/keymap.c (hunk 2 above)
Comment (hlmtre) [reply keyboards/massdrop/alt/keymaps/hlmtre/keymap.c:17]: All done. Apologies, I missed that in my copy.

File: .gitignore (hunk 1 above)
Comment (hlmtre) [reply .gitignore]: Thank you @drashna. I have removed this change in an updated commit. You are correct; it should absolutely be in its own PR.

Comment (hlmtre) [reply .gitignore]: (just saw your guilty spark pfp and i love it)

**Review by drashna — APPROVED** [review]

**Review by tzarc — APPROVED** [review]
